Add Objects (#38)
linuxaustralia/constitution_and_policies: constitution.txt

Note: Schedule 1 of the Act provides that an association’s constitution is to address the association’s financial year.

> 43. Objects of the Organisation

Reviewer: ```suggestion
43. Objects of the association
```

> (1) The objects for which the association is established are:

Reviewer 2: ```suggestion
(1) The objects for which the association is established are: (this is a test)
```

> (1) The objects for which the Organisation is established are:

Reviewer: ```suggestion
(1) The objects for which the association is established are:
```

>   (a) to provide  facilitate communication of information relating to inventions, results, techniques, and governance of open source;

Reviewer: ```suggestion
  (a) to facilitate the communication of information relating to inventions, results, techniques, and governance of open source;
```

>   (b) to support open source projects and the creators of open source ;

Author: ```suggestion
   (b) to support open source projects and the creators of open source;
```

>   (b) to support open source projects and the creators of open source ;

Author: > Everywhere else it's just open source but here it's also open source projects; i

see: https://github.com/linuxaustralia/constitution_and_policies/pull/38#commitcomment-31763437

> Oh, and there's an errant space before the final semi-colon here.

Removed.

>    (b) to support open source projects and the creators of open source;

Author: Change "creators" to the more general term "contributors".

```suggestion
    (b) to support open source projects and the contributors to open source;
```

>     (b) to support open source projects and the contributors to open source;

Author: ```suggestion
   (b) to support open source projects and the contributors to open source;
```

> (nit) some extra spaces here

Fixed.

>   (b) to support the creators of open source and open source projects;

Reviewer: ```suggestion
  (b) to support open source projects and the creators of open source ;
```
Re-ordered the noun phrases to be more clear that we're talking about projects and creators rather than two types of creator.

  (c) to support and promote the adoption of open source;

  (d) to support and promote awareness and understanding of open source;

  (e) to conduct open source advocacy with the government, business and civil society;

  (f) to create a network for related local and online open source communities.

> (2) The Organisation must pursue charitable purposes only and must apply its income in promoting those purposes.

Reviewer: ```suggestion
(2) The association must pursue charitable purposes only and must apply its income in promoting those purposes.
```

> (3) The Organisation must manage payments and gifts to the Organisation and ensure that they are used to further the stated objects of the Organisation.

Reviewer: ```suggestion
(3) The association must manage payments and gifts to the association and ensure that they are used to further the stated objects of the association.
```

> (4) In the event of the Organisation being wound up, its assets must go to a charitable entity whose objectives aren’t inconsistent with the stated objects of the Organisation.

Author: > Should this be association rather than Organisation.

Yep.

```suggestion
(4) In the event of the association being wound up, its assets must go to a charitable entity whose objectives aren’t inconsistent with the stated objects of the association.
```

> (4) In the event of the Organisation being wound up its assets of the must go to a charitable entity whose objectives aren’t inconsistent with the stated objects of the Organisation.

Reviewer: ```suggestion
(4) In the event of the association being wound up its assets of the must go to a charitable entity whose objectives aren’t inconsistent with the stated objects of the association.
```

Reviewer: ```suggestion
(4) In the event of the Organisation being wound up, its assets must go to a charitable entity whose objectives aren’t inconsistent with the stated objects of the Organisation.
```

> (4) In the event of the association being wound up, its assets must go to a charitable entity whose objectives aren’t inconsistent with the stated objects of the association.

Reviewer 3: In the model constitution at:

* https://www.fairtrading.nsw.gov.au/associations-and-co-operatives/associations/starting-an-association/about-the-constitution

The matters regarding winding up the association are part of a separate clause.

Furthermore, it is probably a little too prescriptive. For example, if a creditor (e.g. an employee, the ATO etc) has claims on the assets they **must** receive the assets and it's the assets after the liquidator has ensured a proper liquidation has occurred that gets distributed.

The model rules  suggest - I'd suggest the organisation separates the winding up clause from the objectives (because _winding up_ is not part of its objectives, I should hope) and consider rewording it, perhaps like:

(1)	Subject to the Act and the Regulations, in a winding up of the association, any surplus property of the association is to be transferred to another organisation with similar objects and which is not carried on for the profit or gain of its individual members.
(2)	In this clause, a reference to the surplus property of an association is a reference to that property of the association remaining after satisfaction of the debts and liabilities of the association and the costs, charges and expenses of the winding up of the association.

Author: We've decided to just use the final proposal from last year.  After all the discussion back them there didn't seem to be much point in going through it again.

Reviewer 3: Sure, I get that; just for some background information, when I helped incorporate IT Share SA Inc, (by writing its constitution) and another organisation in South Australia, the (then?) Office of Business Consumer Affairs (the office in the AG's Dept. that handles community associations) the strong response I got back to a clause that sounded like (4) was that it needed to be rewritten because it could have been interpreted as "you shall dispose of the assets to the other not-for-profit-organisation prior to satisfying the debts and liabilities".

IIRC a conflict between the Act and the constitution resolves in favour of the Act.

TL;DR it's probably OK in that, implicitly, it would have to mean any assets remaining after costs, charges, expenses, payment of debt etc.

Author: They are being very picky, as I'm sure it's a given we must operate in accordance with the law and as you say the law says we *must* satisfy creditors before we start giving stuff away.

Anyway, the notice has gone out so it's too late now.  I can only hope the ACNC isn't so picky.  There is some chance because they want the constitution to show we have a charitable purpose, nothing more.

To me it was a major oversight they didn't include disposal of assets on the windup clause, and now you say the model rules the based the constitution on mentions such a clause.  It's all very odd ...

Reviewer 4: Do we have to use the word "fund" because we give grants, or would that be covered under "support"? 
I could find lots of information around accepting grants and the limits around that, but little about giving grants.

Reviewer 5: Is "must" the right word to use in clause 2, 3 and 4, OR, should the word "shall" be used.

Shall is binding, must is, well, not binding.

Reviewer 6: > Shall is binding, must is, well, not binding.

 How do you figure that "must" is not binding? It's pretty absolute (and certainly in [RFC 2119](https://tools.ietf.org/html/rfc2119), which I grant is technical and not legal (and IANAL), they are deemed equivalent: 
> MUST   This word, or the terms "REQUIRED" or "SHALL", mean that the
>    definition is an absolute requirement of the specification.

Reviewer 5: When I working with ISO shall made a clause mandatory/binding, but must didnt. I'm not sure how it relates to legal contexts but at least in ISO/Standards Australia the required term was shall.

RFCs seem more lenient on terminology (which is fine).

All aside, it doesnt change the meaning of the clause and I'm happy either way. Was more an editorial comment than a technical one.

Author: Guys, I know you are getting into practice for the LCA hallway track, but this is set in stone, for now at least.  (You've already had well over a year to comment.)  Voting is open, some have already voted, and changing it after people have cast their votes is not done in polite society.
